packages/elements/src/components/button/__styles__/index.ts

```diff
@@ -51,6 +51,33 @@ export const elButtonSize4 = css`
   }
 `
 
+export const ElButtonLoader = styled.div`
+  @keyframes spinAround {
```

Comment:
Does linaria have a [keyframes](https://styled-components.com/docs/basics#animations) equivalent?
Just thinking here if someone else defines spinAround it'll override this one and vice versa.
If it doesn't then maybe we should centralise our animation definition

Reply:
fair point, I shall have a look in the docs